src/main/java/frc/robot/subsystems/swerve/SwerveContants.java

```diff
@@ -4,15 +4,20 @@
 
 public class SwerveContants {
 
-    public final static double FALCON_MAX_SPEED = 0;
-    public final static double FALCOM_MAX_ANGULAR_VELOCITY = 0;
-    public final static double GEAR_RATIO = 0;
-    public final static double WHEEL_CIRCUMFERENCE = 0;
-
-    public final static Translation2d FRONT_RIGHT_LOCATION = new Translation2d(0, 0);
-    public final static Translation2d FRONT_LEFT_LOCATION = new Translation2d(0, 0);
-    public final static Translation2d BACK_RIGHT_LOCATION = new Translation2d(0, 0);
-    public final static Translation2d BACK_LEFT_LOCATION = new Translation2d(0, 0);
+    public final static double GEAR_RATIO_DRIVE = 6.75;
+    public final static double GEAR_RATIO_ANGLE = 12.6;
+    public final static double WHEEL_CIRCUMFERENCE = 0.314;
+    public final static double WHEEL_RADIUS = 0.05;
+
+
+    // public final static double FALCON_MAX_SPEED = Converstions.falconToMPS(Converstions.RPMToFalcon(6040, GEAR_RATIO_DRIVE), WHEEL_CIRCUMFERENCE, GEAR_RATIO_DRIVE);
+    public final static double FALCON_MAX_SPEED = 2;
+    public final static double FALCOM_MAX_ANGULAR_VELOCITY = 0.1;
+
+    public final static Translation2d FRONT_RIGHT_LOCATION = new Translation2d(0.551942 / 2, 0.551942 / 2);
+    public final static Translation2d FRONT_LEFT_LOCATION = new Translation2d(0.551942 / 2, -0.551942 / 2);
+    public final static Translation2d BACK_RIGHT_LOCATION = new Translation2d(-0.551942 / 2, 0.551942 / 2);
+    public final static Translation2d BACK_LEFT_LOCATION = new Translation2d(-0.551942 / 2, -0.551942 / 2);
```

Comment:
These values come from the length and width of the chassis, right? Then save these as constants, and then also move the Translation2d objects creation to the subsystem code.